core/modules/widgets/blockid.js

```diff
@@ -54,6 +66,14 @@ BlockIdWidget.prototype.hookNavigatedEvent = function(event) {
 	return false;
 };
 
+BlockIdWidget.prototype.hookNavigationAddHistoryEvent = function(event) {
+	// DEBUG: console this._isNavigateToHere(event)
+	console.log(`this._isNavigateToHere(event)`, this._isNavigateToHere(event));
```

Comment:
In the TW core we still use ES5 only. So JS template literals are not possible atm: `` `this._isNavigateToHere(event)` ``
So this code needs to be changed.

Reply:
Sorry, this is debug statement, forget to delete...

Reply:
Created by this in vscode

```
  "Debugger statement": {
    "prefix": "de",
    "scope": "javascript,javascriptreact,typescript,typescriptreact",
    "body": ["// DEBUG: debugger", "debugger;"],
    "description": "Plain debugger statement"
  },
  ```